config/routes.rb

```diff
@@ -1,9 +1,5 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  get 'welcome/index'
```

Comment:
also maybe add route for index. `resources :welcome, only: [:index]` currently you only have `/` path, without full url